src/firmware_main_loop.c

```diff
@@ -39,6 +40,9 @@ void firmware_main_loop(void)
             Abort("NULL workflow spin in main");
         }
         workflow->spin(workflow);
+
+        rust_async_usb_spin();
+
         if (usb_is_enabled()) {
```

Comment:
Also a bit confusing to see `rust_async_usb_spin` regardless of `usb_is_enabled`. As if it were a typo.

Reply:
I see. I guess it could be in that if branch, as indeed no usb ask is spawned without usb :)

Reply:
Fixed.